code/modules/mob/mob_movement.dm

```diff
@@ -88,9 +88,9 @@
 		next_move_dir_sub = 0
 	var/old_move_delay = move_delay
 	move_delay = world.time + world.tick_lag //this is here because Move() can now be called mutiple times per tick
-	if(!mob || !mob.loc)
+	if(!(mob || mob.loc))
 		return FALSE
-	if(!n || !direct)
+	if(!(n || direct))
 		return FALSE
```

Comment:
Shouldn't these be what they were before?
`if(!(n || direct))` would translate to `if(!n && !direct)` if I have done the maths correct

Reply:
Yup, It does the exact same thing this is just a bit neater!

Reply:
Well, that's subjective now that I think about it. I'll revert if you disagree

Reply:
it doesn't do the exact same thing though, it is different
![image](https://user-images.githubusercontent.com/26465327/105629653-3ed30e80-5e3c-11eb-9361-de4e85e455f2.png)

Reply:
oh right. boolean algebra